schemas/com.snowplowanalytics.snowplow/ua_parser/useragent_utils_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
^^ This path is wrong. Can we `git mv` to:

schemas/com.snowplowanalytics.snowplow/user_agent_utils_config/jsonschema/1-0-0